source/usage-examples/find.txt

```diff
@@ -3,3 +3,54 @@ Find Multiple Documents
 =======================
 
 .. default-domain:: mongodb
+
+You can find multiple documents in a collection by using the ``Find()``
+method.
+
+The following example passes a query filter to the ``Find()`` method,
+which matches documents in the ``zips`` collection where the value of the
+``pop`` field is less than or equal to *500*:
+
+.. include:: /includes/run-example-tip.rst
+
+.. literalinclude:: /code-snippets/find.go
+   :start-after: begin find
+   :end-before: end find
+   :emphasize-lines: 4
+   :language: go
+   :dedent:
+
+Click here **<TODO: github link to file>** to see a fully runnable example.
+
+Expected Result
+---------------
+
+This example creates a ``Cursor`` object that returns
+documents similar to these:
+
+.. code-block:: json
+   :copyable: false
+
+   // results truncated
+   ...
+   { ... , "city" : "SHAKTOOLIK", ... , "pop" : 183, "state" : "AK" },
+   { ... , "city" : "DARLINGTON", ... , "pop" : 12, "state" : "ID" },
+   { ... , "city" : "EAST BARRE", ... , "pop" : 381, "state" : "VT" },
```

Comment:
```suggestion
   { ... , "city" : "SHAKTOOLIK", "pop" : 183, "state" : "AK" },
   { ... , "city" : "DARLINGTON", "pop" : 12, "state" : "ID" },
   { ... , "city" : "EAST BARRE", "pop" : 381, "state" : "VT" },
```

[As documents are unordered](https://docs.mongodb.com/manual/reference/operator/aggregation/sort/#sort-consistency), I think it is unnecessary (and potentially implies an order in documents) to include two sets of ellipses.

Reply:
I was 100% wrong! Misread the sorts document. [Key/Values are ordered](https://bsonspec.org/spec.html), documents are unordered. Rea was right! Great job!